src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int top;
+    private int bottom;
+    private int high;
```

Comment:
Consider renaming the variable 'high' to 'height'. This would improve readability and better represent its purpose in the context of an isosceles trapezoid. 📏